subsys/nonsecure/secure_services_ns.c

```diff
@@ -46,3 +46,7 @@ NRF_NSE(void, spm_busy_wait, uint32_t busy_wait_us);
 #ifdef CONFIG_SPM_SERVICE_NS_HANDLER_FROM_SPM_FAULT
 NRF_NSE(int, spm_set_ns_fatal_error_handler, spm_ns_on_fatal_error_t handler);
 #endif /* CONFIG_SPM_SERVICE_NS_HANDLER_FROM_SPM_FAULT */
+
+#if (defined(CONFIG_FPU) != defined(CONFIG_SPM_FPU_CONFIG))
+#error "Mismatch between FPU configuration in APP and SPM"
+#endif
```

Comment:
I think this belongs in the build system, not in the source code.

Reply:
So how do I do that and still get Cmake to work fine?
Ref. Your previous comment: https://github.com/zephyrproject-rtos/zephyr/issues/9573

Reply:
Sorry didn't see your previous comment.

Reply:
Anyways I*m 50/50 split on this. If you don't call into non-secure the configuration shouldn't matter so if you never go from non-secure -> secure through a secure gateway I would assume it would never secure fault. As seen from other samples that may not use non-secure callable functions.